test/unit/app/repos/search.test.js

```diff
@@ -388,7 +388,7 @@ describe('Search repo', () => {
 
       expect(sequelize.transaction).not.toHaveBeenCalled()
       expect(sequelize.models.search_index.create).toHaveBeenCalledWith({
-        search: 'P-123 John Smith org name addr1, addr2, town, postcode',
+        search: 'P-123 John Smith  org name addr1, addr2, town, postcode',
```

Comment:
just double checking that this was purposeful?

Reply:
Yes, if email is blank, it adds an extra space